custom-script/utils.py

```diff
@@ -70,21 +105,20 @@ def parse_trees(trees):
 function evaluatePixel(sample) {{
     let NDWI = index(sample.B03, sample.B08);
     let NDMI = index(sample.B08, sample.B11);
-    
+
     return [predict(sample.B02, sample.B03, sample.B04, NDWI, NDMI)]
 }}
 
 {tree_functions}
 
-function predict({MODEL_INPUTS_STR}) {{ 
+function predict({MODEL_INPUTS_STR}) {{
     return [1/(1+Math.exp(-1*({function_sum})))];
 }}
 """
 
 
-def parse_model(model, js_output_filename=None):
+def parse_model(model: Any, js_output_filename: str = None) -> str:
```

Comment:
`js_output_filename` is clearly not a string, since you assign it the value `None`. Copy over the mypy settings from eo-learn and eo-grow and let mypy help you, that is the whole point of it.

Reply:
done